config/application.rb

```diff
@@ -8,11 +8,5 @@
 
 module ExampleThirdPartyIntegrationApp
   class Application < Rails::Application
```

Comment:
Curious if we need this file anymore 👀

Reply:
Yes we do! This file still contains some important bits:
```
require_relative "boot"
require "rails/all"
Bundler.require(*Rails.groups)
```
and is required in a few other files in the project. I tried removing the require statements and moving the lines above somewhere else, but i got errors because several gems also require the `config/application.rb` file.